README.rst

```diff
@@ -14,103 +14,69 @@ Microsoft Azure SDK for Python
     :target: https://ci.appveyor.com/project/lmazuel/azure-sdk-for-python/branch/master
 
 This project provides a set of Python packages that make it easy to
-access the Microsoft Azure components such as ServiceManagement, Storage\*, and ServiceBus.
+access Management of Microsoft Azure components (Virtual Machines, ...) or Runtime (ServiceBus using HTTP, Batch, Monitor).
```

Comment:
"access Management (Virtual Machines, ...) or Runtime (ServiceBus using HTTP, Batch, Monitor) components of `Microsoft Azure <https://azure.microsoft.com/>`_."